lib/realtime/messages.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Realtime.Messages do
+  @moduledoc """
+  Handles `realtime.messages` table operations
+  """
+  import Ecto.Query
+  alias Realtime.Repo
+  alias Realtime.Api.Message
+
+  @doc """
+  Deletes messages older than 72 hours for a given tenant connection
+  """
+  @spec delete_old_messages(pid()) :: {:ok, any()} | {:error, any()}
+  def delete_old_messages(conn) do
+    limit = NaiveDateTime.utc_now() |> NaiveDateTime.add(-72, :hour)
+    query = from m in Message, where: m.inserted_at <= ^limit
+    Repo.del(conn, query)
```

Comment:
wondering if we want to do a batch delete (a loop with a limit), for example, when needing to delete a large number of records, which can affect the performance of the database

Reply:
good call, might be worth it to explore that 🤔 IMO we can keep as is in this first iteration and check the impact